hyperstart.go

```diff
@@ -430,6 +512,18 @@ func (h *hyper) startOneContainer(pod Pod, c Container) error {
 		return err
 	}
 
+	//TODO : Enter mount namespace
+
+	// Handle container mounts
+	fsmap, err := h.bindMountContainerMounts(pod.id, c.id, c.config.Mounts)
+	if err != nil {
+		h.bindUnmountAllRootfs(pod)
+		return err
+	}
+
```

Comment:
We should call `c.storeMounts()` from here.
I think it should really be stored after we know that we actually modified the container mounts list. Otherwise, doing that into container.go or pod.go is not obvious at all.